package.json

```diff
@@ -76,8 +76,8 @@
     "vite": "^3.1.4"
   },
   "peerDependencies": {
-    "@storybook/svelte": ">=6.4.20",
-    "@storybook/theming": ">=6.4.20",
+    "@storybook/svelte": ">=7.0.0",
+    "@storybook/theming": ">=7.0.0",
```

Comment:
It might be safer to limit this to just `^7`, isn't it?  Case in point, previous versions of this plugin may start to error when used with storybook 7, since it is within their peerDependencies range, but we're making breaking changes to the way things work with the vite builder.

Reply:
yeah it's a matter of temper I think, and what we think about SB 8.0. I've always preferred transitive peer deps to be less restrictive, but I don't care too much. Maybe @shilman have an oppinion? I've made a decision matrix that describes what happens in different SB 8.0 situations:

|  | `>=7.0.0` | `^7.0.0` |
|---|---|---|
| this v3 is supported by SB 8.0 | we and the user don't have to do anything, great! | We have to release a v4 here and the user has to upgrade |
| this v3 is NOT supported by SB 8.0 | SB will break with an error, maybe a bad one, maybe a graceful one. | package manager will complain with unmet peer dependency. We have to release a v4 here and the user has to upgrade |

Reply:
I’m with @IanVS on this. Better safe than sorry and we can always bump in a future version

Reply:
Deal!